src/Analyzers/VisualBasic/Analyzers/ParenthesesForMethodInvocations/VisualBasicParenthesesStyleDiagnosticAnalyzer.vb

```diff
@@ -0,0 +1,71 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.CodeStyle
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.VisualBasic.CodeStyle
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.ParenthesesForMethodInvocations
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Friend NotInheritable Class VisualBasicParenthesesStyleDiagnosticAnalyzer
+        Inherits AbstractBuiltInCodeStyleDiagnosticAnalyzer
+
+        Private Shared ReadOnly s_titleForAddParentheses As New LocalizableResourceString(
+            NameOf(VisualBasicAnalyzersResources.Add_parentheses_to_method_invocation),
+            VisualBasicAnalyzersResources.ResourceManager,
+            GetType(VisualBasicAnalyzersResources))
+
+        Private Shared ReadOnly s_titleForRemoveParentheses As New LocalizableResourceString(
+            NameOf(VisualBasicAnalyzersResources.Remove_parentheses_from_method_invocation),
+            VisualBasicAnalyzersResources.ResourceManager,
+            GetType(VisualBasicAnalyzersResources))
+
+        Protected Sub New()
```

Comment:
Your constructor isn't public. So the type can't be instantiated

Reply:
Oops, that happened when I made the class sealed instead of abstract 😄 
I'll test again with public locally tomorrow, and start working on the code fix & tests.

Thanks for catching this @CyrusNajmabadi